rust_src/src/process.rs

```diff
@@ -318,4 +323,20 @@ pub fn set_process_query_on_exit_flag(process: LispObject, flag: LispObject) ->
     flag
 }
 
+/// Return the exit status of PROCESS or the signal number that killed it.
+/// If PROCESS has not yet exited or died, return 0.
+#[lisp_fn]
+pub fn process_exit_status(process: LispObject) -> LispObject {
+    let p_ref = process.as_process_or_error();
+    if p_ref.raw_status_new() {
+        unsafe { update_status(p_ref.as_ptr()) };
+    }
+    let status = LispObject::from_raw(p_ref.status);
```

Comment:
Out of curiosity, this has worked with `from`, why did we change this again ?

Reply:
I changed what `::from` does; there are some good examples in `README.md`. In this case you can write `pub fn process_exit_status(process: LispProcessRef) -> LispObject` once someone writes an `impl From<LispObject> for LispProcessRef`. It's not a huge benefit for this function, since it immediately has to call an unported function with a raw ptr, but it's a large win overall.